frontend/src/summary_charts.pug

```diff
@@ -27,9 +32,12 @@
         a(
           title="click to view breakdown of commits",
           onclick="deactivateAllOverlays()",
-          v-on:click="openTabZoom(repo[0], filterSinceDate, filterUntilDate, repo, 0)"
+          v-on:click="openTabZoom(repo[0], filterSinceDate, filterUntilDate, true)"
```

Comment:
instead of passing the boolean value here, is it possible to determine it in js?